distributions/otelcol-k8s/postinstall.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if command -v systemctl >/dev/null 2>&1; then
+    systemctl enable otelcol-k8s.service
+    if [ -f /etc/otelcol-k8s/config.yaml ]; then
+        systemctl start otelcol-k8s.service
```

Comment:
Why run it as a journald service? should it be some k8s deployment instead?

Reply:
I've copied this directly from the contrib distro and it is used during the `nfpms` part of the build. Take a look through the `.goreleaser.yaml` file, I suspect we could reduce the number of artifacts we produce for this distro (although it isn't hurting anything to produce them).

Reply:
I think it might be easier to just have those instead of creating rules to skip them